src/ducks/loan/LoanListPage.jsx

```diff
@@ -19,20 +33,22 @@ const DumbLoanListPage = props => {
       {accounts.map(account => (
         <Section key={account._id}>
           <Wrapper>
-            <CompositeRow
-              primaryText={
-                <div className="u-flex u-flex-items-center">
-                  <AccountIcon account={account} />
-                  <Bold className="u-ml-1">{account.label}</Bold>
-                </div>
-              }
-              right={<Icon icon="right" color="var(--coolGrey)" />}
-              actions={<LoanProgress account={account} />}
-              className="u-c-pointer"
-              onClick={() => {
-                filterByDoc(account)
-              }}
-            />
+            <PaddedOnDesktop>
```

Comment:
Mmh, I am getting confused, I thought the Wrapper was supposed to do the padding, but it is late. Anyway, Wrapper is a bit too generic :D